bark-models/pom.xml

```diff
@@ -20,7 +20,7 @@
     <parent>
         <groupId>org.apache.bark</groupId>
         <artifactId>bark-parent</artifactId>
-        <version>1.0-SNAPSHOT</version>
+        <version>0.0.1-SNAPSHOT</version>
```

Comment:
I think we should versioning of 

major.minor.revision

so current release should be 0.1.0-SNAPSHOT instead